wiki/models/platforms-guide.md

```diff
@@ -44,23 +44,42 @@ Enabling the step effect changes the rings spacing when the specified event is c
 ### Tube Light
 ![Tube Light](~@images/models/platforms/TubeLightScript.png)
 
-This script enables blinking lights. Putting this on an empty gameObject changes the background and adds a bit of color
-to that space, according to the light ID's. When there's also a mesh renderer on it, it'll change the meshes color according
-to the light ID's. When using this no color adding is needed, so I change the size on the script to 0.
+This script enables blinking lights.
+Either put this on an empty GameObject and adjust the width, length and center parameters
+or on one with a mesh renderer attached.
+The colors will change according to the specified light ID.
 
 ### Song Events
 ![Song Event Handler](~@images/models/platforms/SongEventHandler.png)
 
-The event manager is the most useful script. With it you can trigger an action on any beat saber event (even unused ones).
-For adding an event press the `+` button underneath `On Trigger ()`. Drag in the object you want to manipulate into the box
-that just got created. Press the dropdown menu to the right and choose what that object should do, by first selecting what
-component, then what action. Make sure that you only use 1 event Handler per gameObject, as only 1 will work per gameObject.
+The `Event Manager` is the most useful script. With it you can trigger an action on any beat saber event (even unused).
```

Comment:
Nitpick: "beat saber" should be capitalized as "Beat Saber" for consistency with the rest of the page.